mrao2/newtowndogs
# login functionality

this is for the login function! it compares the hashed password in the db with the hashed version of the password submitted!

<img width="953" alt="Screenshot 2023-08-15 at 10 17 35 PM" src="https://github.com/mrao2/newtowndogs/assets/110359731/e22fff24-ec51-488b-b480-d416f254605c">
<img width="953" alt="Screenshot 2023-08-15 at 10 18 56 PM" src="https://github.com/mrao2/newtowndogs/assets/110359731/cb663809-352c-4362-a087-66475f1d709b">

To test: 
the dummy account credentials are 
email: dummy@example.com
password: hashed_password

when you inspect and hit login it should log login successful! and redirect to a link to the homepage. in upcoming pulls it will just redirect to the homepage with the pages that should be visible only to authorized users
